addon/components/file-browser-item/template.hbs

```diff
@@ -1,3 +1,3 @@
 {{! TODO: configurable columns }}
 {{file-browser-icon item=item}}
-<a {{action 'open'}}>{{item.itemName}}</a>
+<a {{action 'open'}} role="link">{{item.itemName}}</a>
```

Comment:
Side note: I was under the impression that ARIA naturally provided certain basic roles if you used the right markup.

So for example, an `<a>` would already be understood to be a link. Is that not the case? I'd be curious to know and would be willing to help get started on a screen reader to try it out.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_link_role
> Note: Where possible, it is recommended that you use a native <a> elements rather than the link role, as native elements are more widely supported by older user agents and assistive technology. Native <a> elements also support keyboard and focus requirements by default, without need for additional customization.

http://w3c.github.io/aria-in-html/#rule1
> If you can use a native HTML element [HTML51] or attribute with the semantics and behaviour you require already built in, instead of re-purposing an element and adding an ARIA role, state or property to make it accessible, then do so.